src/olympia/amo/search.py

```diff
@@ -254,18 +254,16 @@ def raw(self):
     def __iter__(self):
         return iter(self._do_search())
 
-    def raw_facets(self):
-        return self._do_search().results.get('facets', {})
+    def raw_aggregations(self):
+        return self._do_search().results.get('aggregations', {})
 
     @property
-    def facets(self):
-        facets = {}
-        for key, val in self.raw_facets().items():
-            if val['_type'] == 'terms':
-                facets[key] = [v for v in val['terms']]
-            elif val['_type'] == 'range':
-                facets[key] = [v for v in val['ranges']]
-        return facets
+    def aggregations(self):
```

Comment:
I would love to see a small test that verifies it works as expected. I don't see any edgecases so it should be reasonable to do.